bin/faucet/Cargo.toml

```diff
@@ -14,7 +14,6 @@ repository.workspace = true
 [features]
 # Makes `make-genesis` subcommand run faster. Is only suitable for testing.
 # INFO: Make sure that all your components have matching features for them to function.
-testing = ["miden-client/testing"]
 
```

Comment:
I think we might still need `testing` feature to make sure faucet uses the same PoW settings as node/client. This feature would probably need to be applied to `miden-objects` and `miden-tx`.

Reply:
Done. Omitted `miden-tx` as it doesn't have a `testing` feature.